composer.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "php/mcrypt",
```

Comment:
There is ongoing discussion about the top level namespace `php` being used on Packagist...

Reply:
Not sure where the discussion is happening, but I hope something in the mix is the idea of using the ‘pecl’ namespace instead of ‘php’.

Reply:
For anyone wondering; the discussion referred to is currently PHP Foundation internal discussion, but suggestions are welcome for consideration! :)

Reply:
see also php/pie#65

Reply:
Updated to use `pecl/mcrypt` package name